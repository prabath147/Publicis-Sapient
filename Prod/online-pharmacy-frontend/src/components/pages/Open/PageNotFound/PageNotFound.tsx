import { Button, Container, Group, Image } from '@mantine/core';
import { Link } from 'react-router-dom';
import Images from '../../../../../src/images/404-error-image.jpg';


export default function PageNotFound() {
  return (
    <Container>
      <Image className="animate__animated animate__tada" style={{ marginTop: 100, width: 350, marginLeft: 'auto', marginRight: 'auto' }}
        src={Images}
      />
      <Group position="center">
        <Link to="/">
          <Button variant="outline" size="sm">
            Take me back to home page
          </Button>
        </Link>
      </Group>
    </Container>
  );
}