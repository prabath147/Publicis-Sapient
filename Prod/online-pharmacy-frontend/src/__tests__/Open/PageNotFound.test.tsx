import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import PageNotFound from '../../components/pages/Open/PageNotFound/PageNotFound';

it('component created', () => {
    const component = renderer.create(
        <BrowserRouter>
            <PageNotFound />
        </BrowserRouter>
    )
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})
