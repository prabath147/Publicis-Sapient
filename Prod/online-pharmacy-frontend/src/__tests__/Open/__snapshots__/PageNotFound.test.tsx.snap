// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component created 1`] = `
<div
  className="mantine-Container-root mantine-15pjuqq"
>
  <div
    className="mantine-Image-root animate__animated animate__tada mantine-15po0m8"
    style={
      Object {
        "height": "auto",
        "marginLeft": "auto",
        "marginRight": "auto",
        "marginTop": 100,
        "width": 350,
      }
    }
  >
    <figure
      className="mantine-11nhzn5 mantine-Image-figure"
    >
      <div
        className="mantine-qqmv3w mantine-Image-imageWrapper"
      >
        <img
          className="mantine-3y8yz3 mantine-Image-image"
          onError={[Function]}
          onLoad={[Function]}
          src="404-error-image.jpg"
          style={
            Object {
              "height": "auto",
              "objectFit": "cover",
              "width": "100%",
            }
          }
        />
      </div>
    </figure>
  </div>
  <div
    className="mantine-Group-root mantine-1u7f61y"
  >
    <a
      className="mantine-gnzaph mantine-Group-child"
      href="/"
      onClick={[Function]}
    >
      <button
        className="mantine-UnstyledButton-root mantine-Button-root mantine-l8qp3y"
        data-button={true}
        type="button"
      >
        <div
          className="mantine-3xbgk5 mantine-Button-inner"
        >
          <span
            className="mantine-qo1k2 mantine-Button-label"
            style={
              Object {
                "textTransform": undefined,
              }
            }
          >
            Take me back to home page
          </span>
        </div>
      </button>
    </a>
  </div>
</div>
`;
